app/utils/lib/RoomTypeConfig.js

```diff
@@ -24,6 +25,20 @@ export const RoomSettingsEnum = {
 	E2E: 'encrypted',
 };
 
+export const RoomMemberActions = {
```

Comment:
For me, it would make more sense if the name of this enum was `RoomActions` since it's not related to members at all. It's only checking the allowed habilities that given room has.

Reply:
sorry but I didn't get your point, all options are related to some user action...

Reply:
I don't agree, it may be something that the system can check to execute some action automatically. This is more related to the room's abilities than what users can do (because that is an extra step that requires permission check.